iset.mm

```diff
@@ -97573,6 +97724,256 @@ Knuth for iterated exponentiation (_Science_ 194, 1235-1242, 1976) and
+     by Jim Kingdon, 21-Feb-2022.) $)
+  sizefiv01gt1 $p |- ( M e. Fin
+      -> ( ( size ` M ) = 0 \/ ( size ` M ) = 1 \/ 1 < ( size ` M ) ) ) $=
+    ( cfn wcel csize cfv cc0 clt wbr c1 w3o cfz co wa simpr wn cn0 wo adantl cz
+    wceq sizecl nn0nlt0 syl adantr pm2.21dd orc fz01or df-3or 3imtr4i 3mix3 0zd
+    nn0zd 1zzd fztri3or syl3anc mpjao3dan ) ABCZADEZFGHZURFTZURITZIURGHZJZURFIK
+    LCZVBUQUSMUSVCUQUSNUQUSOZUSUQURPCVEAUAZURUBUCUDUEVDVCUQUTVAQZVGVBQVDVCVGVBU
+    FURUGUTVAVBUHUIRVBVCUQVBUTVAUJRUQURSCFSCISCUSVDVBJUQURVFULUQUKUQUMURFIUNUOU
+    P $.
+
+  ${
+    $d A m n x $.  $d B m n x $.  $d N x y $.
+    $( The set ` ( 1 ... N ) ` has ` N ` elements.  (Contributed by Jim
+       Kingdon, 21-Feb-2022.) $)
+    sizefz1 $p |- ( N e. NN0 -> ( size ` ( 1 ... N ) ) = N ) $=
+      ( vx vy cn0 wcel c1 cfz co csize cfv cz cv caddc cmpt cc0 cfrec wceq wf1o
+      com syl2anc ccnv cen wbr cuz wf eqid frec2uzf1od f1ocnv f1of 3syl elnn0uz
+      biimpi ffvelrnd frecfzennn ensymd sizeennn wa oveq1 cbvmptv freceq1 ax-mp
+      0zd fveq1i f1ocnvfv2 syl5eqr eqtrd ) ADEZFAGHZIJZABKBLZFMHZNZOPZUAZJZCKCL
+      ZFMHZNZOPZJZAVGVOSEVOVHUBUCVIVTQVGOUDJZSAVNVGSWAVMRZWASVNRWASVNUEVGBOVMVG
+      VBVMUFZUGZSWAVMUHWASVNUIUJVGAWAEZAUKULZUMVGVHVOBVMAWCUNUOCVHVOUPTVGWBWEVT
+      AQWDWFWBWEUQVTVOVMJAVOVMVSVLVRQVMVSQBCKVKVQVJVPFMURUSOVLVRUTVAVCSWAAVMVDV
+      ETVF $.
+
+    $( Two finite sets have the same number of elements iff they are
+       equinumerous.  (Contributed by Jim Kingdon, 21-Feb-2022.) $)
+    sizeen $p |- ( ( A e. Fin /\ B e. Fin ) ->
+                   ( ( size ` A ) = ( size ` B ) <-> A ~~ B ) ) $=
+      ( vn vm vx cfn wcel wa cv cen wbr csize cfv wceq com wrex syl2anc syl cc0
+      wb isfi biimpi adantr ad2antlr simplrl simprl nneneq simplrr enen1 simprr
+      enen2 bitrd cz c1 caddc co cmpt cfrec ensymd sizeennn eqeq12d cuz wf1 0zd
+      wf1o eqid frec2uzf1od f1of1 f1fveq syl12anc 3bitr4rd rexlimddv ) AFGZBFGZ
+      HZACIZJKZALMZBLMZNZABJKZTZCOVMVQCOPZVNVMWCCAUAUBUCVOVPOGZVQHZHZBDIZJKZWBD
+      OVNWHDOPZVMWEVNWIDBUAUBUDWFWGOGZWHHZHZVPWGJKZVPWGNZWAVTWLWDWJWMWNTVOWDVQW
+      KUEZWFWJWHUFZVPWGUGQWLWAVPBJKZWMWLVQWAWQTVOWDVQWKUHZAVPBUIRWLWHWQWMTWFWJW
+      HUJZBWGVPUKRULWLVTVPEUMEIUNUOUPUQSURZMZWGWTMZNZWNWLVRXAVSXBWLWDVPAJKVRXAN
+      WOWLAVPWRUSEAVPUTQWLWJWGBJKVSXBNWPWLBWGWSUSEBWGUTQVAWLOSVBMZWTVCZWDWJXCWN
+      TWLOXDWTVEXEWLESWTWLVDWTVFVGOXDWTVHRWOWPOXDVPWGWTVIVJULVKVLVL $.
+  $}
+
+  ${
+    $d A f $.  $d B f $.
+    $( The size of two finite sets is equal if and only if there is a bijection
+       mapping one of the sets onto the other.  (Contributed by Alexander van
+       der Vekens, 17-Dec-2017.) $)
+    sizeeqf1o $p |- ( ( A e. Fin /\ B e. Fin )
+        -> ( ( size ` A ) = ( size ` B ) <-> E. f f : A -1-1-onto-> B ) ) $=
+      ( cfn wcel wa csize cfv wceq cen wbr cv wf1o wex sizeen bren syl6bb ) ADE
+      BDEFAGHBGHIABJKABCLMCNABOABCPQ $.
+
+    $( There is no bijection between a finite set and an infinite set.  By
```

Comment:
Can you prove "there is no surjection" ?

Reply:
Dunno. set.mm just seems to have the bijection case as far as I've noticed so far anyway.